deploy/chart/values.yaml

```diff
@@ -28,7 +28,7 @@ catalog:
     beta.kubernetes.io/os: linux
 
 package:
-  replicaCount: 1
+  replicaCount: 2
```

Comment:
We'll have to make a new release for these to take effect, right @njhale.

Reply:
Yes, just run `make release` - #712 needs to be merged immediately and has changes to template and manifest prefixes, so when you rebase you'll need to update these again.